### ISSUE TITLE
Review the list of asciidoctor extensions lib stored in the project

### ISSUE THREAD

**Issue by tbouffard** [bonitasoft/bonita-documentation-site#231]
In commit b73ff82, we have
- inline-man-macro.js (may come from https://gitlab.com/antora/antora-asciidoctor-extensions). Contains ref to `couchbase-cli-`
- json-config-ui-block-macro.js
- multirow-table-head-tree-processor.js
- source-url-include-processor.js:
  - do not work with Antora 3, see #266. 
- swagger-ui-block-macro.js: we don't use swagger in our documentation
- tabs-block.js (may come from https://gitlab.com/antora/antora-asciidoctor-extensions)

One of the file suggests that they have been copied from the couch base Antora repository that hosts the same files
"const BUNDLE_BASE_URL = 'https://couchbase'"

All of them have been imported at the repository creation. AFAIK, we don't know if we need them to build the documentation.
We should review them, keep only the required ones, document where they come from, why we need them, ... as they may introduce issues with the upcoming Antora 3.0 version #229
**--> none of them is required for the doc-site generation**

See also https://asciidoctor.org/docs/extensions or https://docs.asciidoctor.org/asciidoctor.js/latest/extend/extensions/ecosystem/

**Comment by tbouffard** [comment]
### About the `tab-blocks` extension

Taking the example from the extension source code, the html is correctly generated, but the rendering is not accurate as we don't define the CSS classes in the theme

![image](https://user-images.githubusercontent.com/27200110/151496418-0ea28433-a482-4ff3-b721-95416f662b31.png)


```html
<div class="sectionbody">
<div class="tabset is-loading">
<div class="ulist tabs">
<ul>
<li>
<p><a id="tabset1_tab-a"></a>Tab A</p>
</li>
<li>
<p><a id="tabset1_tab-b"></a>Tab B</p>
</li>
</ul>
</div>
<div class="content">
<div class="tab-pane" aria-labelledby="tabset1_tab-a">
<div class="paragraph">
<p>Contents of tab A.</p>
</div>
</div>
<div class="tab-pane" aria-labelledby="tabset1_tab-b">
<div class="paragraph">
<p>Contents of tab B.</p>
</div>
</div>
</div>
</div>
</div>
```

**Comment by tbouffard** [comment]
Closed by #296
